prow/cmd/peribolos/main.go

```diff
@@ -675,6 +721,11 @@ func configureTeam(client editTeamClient, orgName, teamName string, team org.Tea
 	} else {
 		gt.Privacy = ""
 	}
+	if parent > 0 && gt.ParentTeamID != &parent {
```

Comment:
Can I make a child team into a top-level team by later setting its parentID to 0? I think so, in which case we probably don't need the `parent > 0` check?

Reply:
no.. you actually need to send `parent_team_id: null` for that to work. removed the omitempty so it will handle this case.